dts/msm8916/msm8939-mtp.dts

```diff
@@ -16,8 +16,22 @@
 	};
 
 	xiaomi-ferrari {
+		/* Included becuase log said qcom,msm-id entry not found and this made it recognize the device tree correctly */
+		qcom,msm-id = <239 0>;
+		qcom,board-id = <8 0>;
+		
 		model = "Xiaomi Mi 4i";
 		compatible = "xiaomi,ferrari", "qcom,msm8939", "lk2nd,device";
-		lk2nd,match-cmdline = "* mdss_mdp.panel=1:dsi:0:qcom,mdss_dsi_sharp_rsp61322_1080p_video:1:none";
```

Comment:
Need to replace this with `lk2nd,match-panel;`

Reply:
Ok